pkg/cmd/roachtest/cdc.go

```diff
@@ -907,8 +916,14 @@ if ! [[ -f "$CONFLUENT_TAR_PATH" ]] || ! checkFile "$CONFLUENT_TAR_PATH" "$CONFL
     sleep 15;
   done
 fi
+
 tar xvf /tmp/confluent.tar.gz -C "$CONFLUENT_DIR"
-`
+
+cd "${CONFLUENT_DIR}/${CONFLUENT_INSTALL_BASE}"
+curl --retry 3 --fail --show-error -L --http1.1 https://cnfl.io/cli | sh -s -- -b bin/
```

Comment:
This is unfortunately the official way to install this tool from what I can see. I'm looking into whether the license allows us to cache it in gcs